reinforced_lib/rlib.py

```diff
@@ -1,6 +1,10 @@
+from __future__ import annotations
```

Comment:
Korzystamy w jakiś sposób z pakietu `annotations`? Bo nie widzę tego w kodzie, a zostawianie nieużywanych importów z `__future__` może być nieeleganckie (i ryzykowne jeśli coś się zmieni w tym pakiecie w przyszłości).

Reply:
Tak, jest to niezbędne, ponieważ metoda `load()`, która należy do klasy RLib, zwraca instancję RLib. Bez tego importu python nie pozwalał mi na anotację `load() -> RLib`. A ta anotacja jest bardzo przydatna, bo dzięki niej użytkownik ma typehinting dla metod tej klasy.